### ISSUE TITLE
Using UIAppearance prevents dynamic changes

### ISSUE THREAD

**Issue by phedlund** [nicolaschengdev/WYPopoverController#39]
Thanks for a great control. Changes to UIAppearance do unfortunately not apply to existing instances of a control. Thus it is currently not possible to change the colors of a popover while it is shown.

**Comment by nicolaschengdev** [comment]
No you cannot change appearance properties like colors when popover is shown.

Perhaps, in a next release, I will add a _style_ named property on `WYPopoverController` which will be used like a proxy in order to achieve this goal.

**Comment by phedlund** [comment]
Maybe the easiest would be to expose the containerView as a property?

**Comment by nicolaschengdev** [comment]
The popover is composed with two views :
- `containerView` as background view 
- `innerView` as foreground view

These two views share few properties to render the popover.
Manipulate these two views is a bit  complicated.

That's why I think a new property named _style_ is required in order to keep things simple to use.

**Nicolas**

**Comment by phedlund** [comment]
Sounds great. The reason I want this is to be able to dynamically switch "themes", actually between "day" and "night" modes in a reader app.

**Comment by nicolaschengdev** [comment]
fixed issue #39, see commit [d7fab2e644...](https://github.com/nicolaschengdev/WYPopoverController/commit/d7fab2e64462ae60ba4392f5290a1effaeabfde4)

Introduced in 0.2.1 release, you can change appearance of your popovers with themes.

``` objective-c
- (void)changePopoverTheme
{
    popover.theme = [WYPopoverTheme themeForIOS6];  // you set a new theme

    // use beginThemeUpdates and endThemeUpdates methods if you have to change several values which compose your current theme

    [popover beginThemeUpdates];
        popover.theme.arrowHeight = 30;
        popover.theme.arrowBase = 40;
    [popover endThemeUpdates];
}
```
